### DISCUSSION TITLE
Resetting system to zero logs

### DISCUSSION THREAD

**Discussion by kvkamin** [Security-Onion-Solutions/securityonion#13603] [Unsupported Versions] [answered]
Is it possible to execute so-nsm-clear and so-elastic-clear without removing Elastic Agent registrations?

**Comment by reyesj2** [comment, marked as answer]
Yes, running so-nsm-clear and so-elastic-clear can be run without removing elastic agent registrations. I went ahead and ran it on my grid and still have two linux boxes connected to fleet.

**Reply by kvkamin** [reply]
thanks!
